js/src/wp-seo-post-scraper.js

```diff
@@ -1,9 +1,13 @@
-/* global YoastSEO, tinyMCE, wp, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console */
+/* global YoastSEO, tinyMCE, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console, require */
 (function( $ ) {
 	'use strict';
 
 	var currentKeyword = '';
 
+	// Initialize the keyword tab module.
+	var keywordTab;
```

Comment:
If I understand correctly, the difference is the capitalized 'k' in KeywordTab and keywordTab. It's easy to mix those two up, so I would advise to change the name of one of the vars.